### ISSUE TITLE
Recently updated to 14.2.6 from 12 and now I get a Thread 1: "No active configuration.  Make sure GIDClientID is set in Info.plist."

### ISSUE THREAD

**Issue by ColtonIdle** [firebase/FirebaseUI-iOS#1215]
### Step 1: Are you in the right place?
yes? (hopefully im new to ios dev so im not sure if this is a bug in the lib or a setup issue)

### Step 2: Describe your environment
spm 14.2.6
  
### Step 3: Describe the problem:
No active configuration.  Make sure GIDClientID is set in Info.plist.
  
#### Relevant Code:
To support the update from 12 to latest stable I had to change

GIDSignIn.sharedInstance.signIn(with: signInConfig, presenting: viewController)

to 

GIDSignIn.sharedInstance.signIn(presentingWith: viewController)

and I'm assuming that's why I get No active configuration.  Make sure GIDClientID is set in Info.plist.

**Comment by ColtonIdle** [comment]
Seems like I just have to call

GIDSignIn.sharedInstance.configuration = config ? and it works?

**Comment by morganchen12** [comment]
Yeah, the GoogleSignIn SDK will look for your `GIDClientID` in the Info.plist and in Firebase's GoogleService-Info.plist. If the value isn't set in either of those you need to set it programmatically.
